src/dune/dune_file.ml

```diff
@@ -971,6 +971,10 @@ module Executables = struct
       let stanza = pluralize stanza ~multi in
       let names =
         let open Dune_lang.Syntax.Version.Infix in
+        Option.iter names
+          ~f:
+            (List.iter ~f:(fun name ->
+                 ignore (Module_name.parse_string_exn name)));
```

Comment:
Can we specify the type of the value we're ignoring? That's a convention we follow in dune because it's error prone to keep ignoring values if the type returned changes.